OHIF/Viewers
# feat(thumbnail highlight): Thumbnails of hydrated series are now highlighted

### Context

Thumbnails representing hydratable series (i.e. SR, SEG, RTSTRUCT, etc.) are now highlighted to indicate when the item is hydrated in OHIF. 
Addresses OHIF issue #3421.
Also, thumbnails with no image that are 'active' (i.e. the viewport is selected in the viewer) are now also highlighted appropriately with a border just like the image thumbnails.
The highlighting is per as described [here](https://github.com/OHIF/Viewers/issues/3421#issuecomment-1632988610)



### Changes & Results

SEG and RTSTRUCT display sets now have their `isHydrated` flag set to true when they are hydrated. Hydrated SR display sets for tracked measurements already had this flag set appropriately.

The `ThumbnailNoImage` modality label colours and the "list-bullets" icon color when the thumbnail is hydrated are now set based on whether the associated display set is hydrated or not. Also a change was made in `ThumbnailNoImage` so that it indicates an active series (i.e. when visible and highlighted in an OHIF viewport) with a border just like the `Thumbnail` (with an image).



### Testing

A few test cases to consider...

Test A
1. View a study with tracked measurements.
2. Double click or drag and drop the SR into a viewport.
3. Opt NOT to track the measurements and simply view the SR in the viewport.
4. The thumbnail for the SR should have a border indicating that it is the active viewport. Activating a different viewport should remove the border around the thumbnail.
![image](https://github.com/OHIF/Viewers/assets/109477394/6d16c9b3-f56a-43ea-90b2-8e36ee839edc)

5. Similar behaviour should occur when loading a  study with SEG or RTSTRUCT and opting NOT to hydrate the segmentation.

Test B
1. View a study with tracked measurements.
2. Double click or drag and drop the SR into a viewport.
3. Opt to track the measurements.
4. The thumbnail for the hydrated SR should look similar to below...
![image](https://github.com/OHIF/Viewers/assets/109477394/9047c539-b51e-416f-a9b2-f9ef6109bcad)

5. Similar behaviour should occur when loading a  study with SEG or RTSTRUCT after opting to hydrate the segmentation.



### Checklist

#### PR



  semantic-release format and guidelines.

#### Code

  etc.)

#### Public Documentation Updates



  additions or removals.

#### Tested Environment

- [x] Browser: Chrome 115.0.5790.171
  


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
